src/backend/access/external/pxffilters.c

```diff
@@ -187,13 +200,18 @@ Oid pxf_supported_types[] =
 	BYTEAOID,
 	BOOLOID,
 	DATEOID,
-	TIMESTAMPOID
+	TIMESTAMPOID,
+	/* complex datatypes*/
+	INT2ARRAYOID,
+	INT4ARRAYOID,
+	INT8ARRAYOID,
+	BOOLARRAYOID,
+	TEXTARRAYOID
```

Comment:
Can you check what ANYARRAYOID means ? If we can support this datatype as well along with the other array datatypes